### ISSUE TITLE
[Query] Is it MiniSearch could not index frontmatter fields?

### ISSUE THREAD

**Issue by HelloYu** [vuejs/vitepress#3083]
I have some markdown files transform from WordPress, only have frontmatter title field, in markdown content do not have `# h1` mark,
and I found out the MiniSearch could not search by frontmatter title, and I read some VitePress and MiniSearch docs, didn't find any useful information, anyone can tell me, how can I configuring to support this feature?

**Comment by brc-dd** [comment]
will be fixed by #3032

**Comment by brc-dd** [comment]
Refer https://vitepress.dev/reference/default-theme-search#example-transforming-content-adding-anchors

**Comment by HelloYu** [comment]
@brc-dd Thanks! you save my day, i don't read this page, even I went to read the VitePress source code.

**Comment by renqian805** [comment]
> Refer https://vitepress.dev/reference/default-theme-search#example-transforming-content-adding-anchors

[Reproduction](https://stackblitz.com/edit/vite-urdab2?file=docs%2Fexample.md&file=docs%2F.vitepress%2Fconfig.ts)

It appears that it cannot fully solve my problem.

<img width="1683" alt="image" src="https://github.com/vuejs/vitepress/assets/84910084/461e2a75-96ae-45c0-8433-4290386653d3">

**Comment by HelloYu** [comment]
> > Refer https://vitepress.dev/reference/default-theme-search#example-transforming-content-adding-anchors
> 
> [Reproduction](https://stackblitz.com/edit/vite-urdab2?file=docs%2Fexample.md&file=docs%2F.vitepress%2Fconfig.ts)
> 
> It appears that it cannot fully solve my problem.
> 
> <img alt="image" width="1683" src="https://user-images.githubusercontent.com/84910084/274922011-461e2a75-96ae-45c0-8433-4290386653d3.png">

#3024 not release yet.

**Comment by brc-dd** [comment]
Yeah that would be fixed with #3032 (#3024). The solution I mentioned only helps if you're adding frontmatter title via layout.
